data-raw/wd_pcon_lad_la_rgn_ctry.R

```diff
@@ -0,0 +1,123 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Created: 05/08/2024, Cam Race
+#
+# This script creates the wd_pcon_lad_la_reg_ctry data set
+#
+# If updating the data, follow these steps:
+# 1. Load the package using `devtools::load_all(".")`
+# 2. Add in an extra year for the new year into this script within the
+#    create_timeseries_lookup function (R/datasets_utils.R)
+# 3. Check if we need to add a new year into the LAD to region lookup
+# 4. Add the extra year to descriptions, params and validation code in
```

Comment:
Feel like this doesn't quite square up with what needs updating. As far as I can see it's more a case of updating "the options listed for the year @param in the fetch.R @description"?

The validation doesn't look like it validates against the available years (although I was thinking of suggesting it could do...).

Reply:
Fair point! Done

Reply:
Not done any validation on years, purely as that's an extra bit of code that we'll have to maintain and change each time, so not sure if it's worth it?